helm-charts/splunk-otel-collector/templates/securityContextConstraints.yaml

```diff
@@ -11,37 +11,6 @@ metadata:
     heritage: {{ .Release.Service }}
 users:
 - system:serviceaccount:{{ .Release.Namespace }}:{{ template "splunk-otel-collector.serviceAccountName" . }}
-priority: 10
```

Comment:
the helpers are pretty crowded and I think it's a best practice to keep this content here: https://helm.sh/docs/chart_best_practices/templates/